definition/data/sheets/AuthorisationCaseField.json

```diff
@@ -165,5 +166,6 @@
   {"CaseTypeID": "MoneyClaimCase", "CaseFieldID": "SOTLabel", "UserRole": "letter-holder", "CRUD": "CRUD"},
   {"CaseTypeID": "MoneyClaimCase", "CaseFieldID": "submitterLabel", "UserRole": "letter-holder", "CRUD": "CRUD"},
   {"CaseTypeID": "MoneyClaimCase", "CaseFieldID": "caseDocuments", "UserRole": "letter-holder", "CRUD": "CRUD"},
-  {"CaseTypeID": "MoneyClaimCase", "CaseFieldID": "migratedFromClaimStore", "UserRole": "letter-holder", "CRUD": "CRUD"}
+  {"CaseTypeID": "MoneyClaimCase", "CaseFieldID": "migratedFromClaimStore", "UserRole": "letter-holder", "CRUD": "CRUD"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseFieldID": "claimSubmissionOperationIndicators", "UserRole": "letter-holder", "CRUD": "CRUD"}
```

Comment:
Don't we need to add entry of `claimSubmissionOperationIndicators ` for `caseworker-cmc` ?

Reply:
caseworker-cmc , will he create a case in CCD?

Reply:
yes, at some point using CCD UI after inversion of control. Currently are maintaining the definition inline with that for all other fields.

Reply:
have added now for consistency. Done

Reply:
Thanks 👍